data/metadata.json

```diff
@@ -83,6 +83,18 @@
     "clubhouse/dialog/open": {
         "sound-file": "clubhouse/dialog_popup.webm"
     },
+    "clubhouse/ending/episode1": {
+        "sound-file": "clubhouse/ending/episode1.webm",
+        "loop": true
```

Comment:
It looks like these should probably have `"type": "bg"` or else they will play over any background music that is already playing.

Reply:
I've just added the "bg" type.